core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -190,27 +190,6 @@ public Response createFile(@QueryParam("path") String path,
   }
 
   /**
-   * @summary get the list of file block descriptors for a file
```

Comment:
since REST API is being introduced in 1.1, there is no point introducing deprecated endpoints